docs/faq.md

```diff
@@ -79,11 +93,11 @@ jupytext --set-formats ipynb,py:light notebook.ipynb   # Pair a notebook to a li
 jupytext --sync notebook.ipynb                         # Sync the two representations
 ```
 
-## If only I had known of Jupytext before!
+## Can I re-write my git history to use text files instead of notebooks?
 
-Do you feel like rewriting the history of your repository and replacing every `.ipynb` file with its Jupytext Markdown representation? Technically that's just a matter of executing:
+Indeed! You can substitute every `.ipynb` file in the project history with its Jupytext Markdown representation using e.g.:
```

Comment:
Isn't this something of a nuclear option, removing the ipynb files? Maybe also handy to provide a way to just move / stash the processed `.ipynb` files somewhere?

Reply:
Sure! And also rewriting the history is something that one should not do too often... Maybe I'll just mention that as an fun exercise 😄

Reply:
The rewriting history thing is absolutely brilliant though! :-)